index.yaml

```diff
@@ -12,13 +12,13 @@ entries:
       catalog.cattle.io/release-name: fleet
     apiVersion: v2
     appVersion: 0.3.6-rc2
-    created: "2021-06-16T21:05:17.210021-07:00"
+    created: "2021-06-18T00:32:49.943823-07:00"
     dependencies:
     - condition: gitops.enabled
       name: gitjob
       repository: file://./charts/gitjob
     description: Fleet Manager - GitOps at Scale
-    digest: e0851254ef96f419c2d2bd0976ffe1651e85b57a3a3c895ae83c7b3df972e09f
+    digest: 40e7caa239309b735cda4973d199b9e57c7967924395e35defe7baa14f9f82de
```

Comment:
fleet is not touched in this PR. I guess the digest was not updated correctly last time fleet was updated.

Reply:
Is fleet staying separate ? I think we went back and forth. We can do that in follow up either way, but lets ask.